cmd/authelia-suites/main.go

```diff
@@ -83,7 +83,7 @@ func setupSuite(cmd *cobra.Command, args []string) {
 
 	suiteResourcePath := cwd + "/internal/suites/" + suiteName
 
-	exist, err := utils.FileExists(suiteResourcePath)
+	exist, err := utils.PathExists(suiteResourcePath)
```

Comment:
Not sure how this got here. Thinking a refactoring bug?

Reply:
Never mind I refactored FileExists to specifically check if a file exists rather than any path.